src/scripts/components/App.js

```diff
@@ -35,8 +35,13 @@ const PortalPlayWrapper = styled.div`
 `
 
 class App extends Component<Props, void> {
+  constructor (props: Props) {
+    super(props)
+
+    this.props.initializeApp()
+  }
+
   render () {
-    setupKeystore()
```

Comment:
Not good practice to call code w/ side-effects on every render. Better to put this above in the constructor